communitydelphi.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Code Analyzer for Delphi
 homepageUrl=https://github.com/integrated-application-development/sonar-delphi
-archivedVersions=1.0.0,1.1.0,1.2.0,1.3.0,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0
-publicVersions=1.11.0
+archivedVersions=1.0.0,1.1.0,1.2.0,1.3.0,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0,1.11.0
+publicVersions=1.12.0
 
 defaults.mavenGroupId=au.com.integradev.delphi
 defaults.mavenArtifactId=sonar-delphi-plugin
 
+1.12.0.description=Improvements to intrinsics, parsing improvements, bug fixes
+1.12.0.sqVersions=[9.9,LATEST]
```

Comment:
With the release of SonarQube Community Build and SonarQube Server, this changes to:
```
1.12.0.sqs=[10.8,LATEST]
1.12.0.sqcb=[24.12,LATEST]
1.12.0.sqVersions=[9.9,10.7]
```

Reply:
Thanks, I've updated to the new format.